spec/lib/traffic_source_parser/parser/utmz_parser_spec.rb

```diff
@@ -54,42 +54,42 @@
 
       end
 
-      # context 'when value is from referral' do
-      #   {
-      #     cookie_value: "210677130.1432831711.1.1.utmcsr=rakuten|utmc" +
-      #                   "cn=linkshare|utmcmd=(not set)",
-      #     source: "rakuten",
-      #     campaign: "linkshare",
-      #     channel: "Referral",
-      #     medium: "(not set)"
-      #   }
-      # end
-      #
-      # context 'when value is from other advertising' do
-      #   {
-      #     cookie_value: "10083233.1432828147.6.5.utmcsr=newsletter|ut" +
-      #                   "mccn=20150527|utmcmd=newsletter-ef|utmctr=communi" +
-      #                   "tycolleges|utmcct=programa",
-      #     source: "newsletter",
-      #     campaign: "20150527",
-      #     channel: "Other",
-      #     medium: "newsletter-ef",
-      #     term: "communitycolleges",
-      #     content: "programa"
-      #   }
-      # end
-      #
-      # context 'when value is from CPC campaign' do
-      #   {
-      #     cookie_value: "63514687.1432831892.1.1.utmcsr=adwords_gereb" +
-      #                   "oletos1min5_22072014|utmgclid=CNjQrtjy5MUCFQYXHwo" +
-      #                   "d7k0A8g|utmccn=(not set)|utmcmd=(not set)",
-      #     source: "adwords_gereboletos1min5_22072014",
-      #     campaign: "(not set)",
-      #     channel: "Paid Search",
-      #     medium: "cpc"
-      #   }
-      # end
+      context 'when value is from referral' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.